client/src/app/shared/shared-main/account/actor-avatar-info.component.scss

```diff
@@ -70,3 +70,16 @@
     }
   }
 }
+
+::ng-deep .popover-avatar-info .popover-body {
```

Comment:
Please scope `::ng-deep` using a parent select (for example: `.actor-img-edit-container ::ng-deep ...`)